addons/crm/views/crm_lead_views.xml

```diff
@@ -332,7 +332,58 @@
                     <field name="message_needaction" invisible="1"/>
                     <field name="tag_ids" optional="hide" widget="many2many_tags" options="{'color_field': 'color'}"/>
                     <field name="priority" optional="hide"/>
-                    <field name="activity_exception_decoration" widget="activity_exception"/>
+                </tree>
+            </field>
+        </record>
+
+        <record id="crm_case_tree_view_my_activities" model="ir.ui.view">
```

Comment:
Not sure why there are so much new views... I feel like only a new tree view is necessary. It could be based on existing view, using inherit_id (inherit from the based one) + mode = primary (becomes independent).

Inherit + primary allows to somehow "duplicate" a view: it became an independent view, still inheriting from the base one. It means that fields added to the base one are added to the inheriting one. You can then tweak your custom one with xpath to hide or remove unnecessary fields / add relevant one.

Also don't forget to give an higher priority for this view. Purpose is to avoid having it displayed by default (lower priority means default).

You can check for mode="primary" in the code to see examples :)